module/modmanager/manager.go

```diff
@@ -225,6 +225,7 @@ func (mgr *Manager) Add(ctx context.Context, confs ...config.Module) error {
 		go func(i int, conf config.Module) {
 			defer wg.Done()
 
+			mgr.logger.CInfow(ctx, "now adding module", "module", conf.Name)
```

Comment:
We do not have/I didn't add a corresponding log for the "success" of these additions/reconfigurations/removals. Do we want those? Or is the absence of an error log after these "now adding/reconfiguring/removing" logs good enough. Note that the I do `Info` log successful connections to remotes (that log was existing and I just bumped it to `Info`).